src/lib/Libwin/dir.c

```diff
@@ -152,7 +156,9 @@ closedir(DIR *hdir)
 		return (-1);
```

Comment:
did you miss checking and loggin for error at line 124/128 for api FindNextFile() ?